tx.go

```diff
@@ -101,22 +108,33 @@ func (b *Tx) ResolveLink(p []string) (*node.Link, []string, error) {
 	return lnk, rest, nil
 }
 
-func (b *Tx) Size() (uint64, error) {
-	return uint64(b.tx.Size().Int64()), nil
+// String is a helper for output
+func (t *Tx) String() string {
+	return fmt.Sprintf("<EthereumTx %s>", t.Cid())
 }
 
-func (b *Tx) Stat() (*node.NodeStat, error) {
-	return &node.NodeStat{}, nil
+// Tree lists all paths within the object under 'path', and up to the given depth.
+// To list the entire object (similar to `find .`) pass "" and -1
+func (t *Tx) Tree(p string, depth int) []string {
+	return []string{"toAddress", "value", "data", "nonce", "gasPrice", "gas"}
```

Comment:
Technically, this should check if depth is 0 and return `[]string{}` in that case (not your change but we might as well fix this).